v2/instructor-client/src/Course.js

```diff
@@ -13,7 +14,7 @@ export default function Course({ cid, role, minimizeStatus, creationTime, fetchC
 	const [course, setCourse] = useState(null);
 	const [loading, setLoading] = useState(true);
 	const [managing, setManaging] = useState(false);
-	const [renaming, setRenaming] = useState(false);
+	const [show, setShow] = useState(false);
```

Comment:
Please indicate in the variable name which modal this is related to (e.g. showEditCourseDefault)